components/ActionsList/index.js

```diff
@@ -0,0 +1,48 @@
+import DueDate from './DueDate';
+import ExpandingText from 'components/ExpandingText';
+import Heading from 'components/Heading';
+import Table, {
+  TableHead,
+  TableRow,
+  TableHeader,
+  TableBody,
+  TableData
+} from 'components/Table';
+
+const ActionsList = ({ actions }) => (
```

Comment:
would we call gateway from pages/plans/[id] to get the plans? What's the benefit of passing in actions here instead of passing the ID and getting the actions from within here?

Reply:
We fetch the plan and all the actions on page load at the moment, so adding a fetch in at the component level would end up fetching everything twice?

Reply:
yes. That makes a lot of sense